
dict: {'num_obstacles': 5, 'object_size': 20, 'window_size': 512, 'action_step_scaling': 20, 'obstacle_step_scaling': 5, 'delta_T': 1, 'render_fps': 24, 'input_dims': 26, 'batch_size': 4096, 'gamma': 0.999, 'target_update': 10, 'alpha': 0.0003, 'beta': 0.0003, 'tau': 0.005, 'entropy_factor': 0.5, 'entropy_factor_final': 0.5, 'num_episodes': 100, 'sigma_init': 2.0, 'sigma_final': 2.0, 'pretrain': True, 'num_episodes_pretrain': 100, 'maxsize_ReplayMemory': 100000, 'action_smoothing': True, 'action_history_size': 3, 'select_action_filter': True, 'select_action_filter_after_episode': 70, 'sort_obstacles': True, 'apply_environment_seed': True, 'seed_init_value': 3407, 'plot_durations': True, 'plot_sigma': True, 'scheduler_period': 2}
/home/volkan/anaconda3/envs/adlr/lib/python3.9/site-packages/gym/spaces/box.py:127: UserWarning: [33mWARN: Box bound precision lowered by casting to float32
  logger.warn(f"Box bound precision lowered by casting to {self.dtype}")
Traceback (most recent call last):
  File "/home/volkan/31_Projects/adlr/tum-adlr-09/SAC-X/training.py", line 459, in <module>
    _, reward, done, _, _ = env.step(action)
  File "/home/volkan/31_Projects/adlr/tum-adlr-09/SAC-X/environment.py", line 350, in step
    last_x_steps.append(last_x_positions[i] - last_x_positions[i - 1])
IndexError: deque index out of range
Testing random seed: tensor([0.6346, 0.6923])